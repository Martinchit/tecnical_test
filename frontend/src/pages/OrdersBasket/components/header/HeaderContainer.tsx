import styled from 'styled-components';
import { FlexBox } from '../../../../core/components/atoms/FlexBox';

export const HeaderContainer = styled(FlexBox)`
  justify-content: flex-end;
  margin: 2% 0;
  width: 100%;

  @media (max-width: 767px) {
    justify-content: center;
  }
`;
