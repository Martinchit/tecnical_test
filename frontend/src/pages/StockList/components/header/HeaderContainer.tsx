import styled from 'styled-components';
import { FlexBox } from '../../../../core/components/atoms/FlexBox';

export const HeaderContainer = styled(FlexBox)`
  justify-content: flex-end;
  margin: 1% 0;
  width: 100%;

  @media (max-width: 767px) {
    flex
    justify-content: center;
    flex-direction: column;
    margin: 5% 0;
  }
`;
