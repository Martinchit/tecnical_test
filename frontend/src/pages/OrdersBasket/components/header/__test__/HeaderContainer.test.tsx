import React from 'react';
import { shallow } from 'enzyme';

import { HeaderContainer } from '../HeaderContainer';

describe('HeaderContainer', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<HeaderContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
