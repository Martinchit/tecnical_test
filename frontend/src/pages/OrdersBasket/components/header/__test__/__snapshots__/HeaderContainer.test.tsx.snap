// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderContainer renders correctly 1`] = `
<div
  className="sc-AxjAm sc-AxirZ fShRZP"
/>
`;
